docs/accelerators.md

```diff
@@ -87,13 +87,17 @@ docker run --rm -it  --device=/dev/dri --group-add=$(stat -c "%g" /dev/dri/rende
 ```
 
 >Note:
-> The public docker image includes the OpenCL drivers for GPU in version 21.38.21026.
-To use an older version, build the image with the following parameter: 
+> The public docker image includes the OpenCL drivers for GPU in version 21.38.21026 (RedHat) and 21.48.21782 (Ubuntu).
 
+To use an older version, build the image with the following parameter: 
```

Comment:
I think we could drop from readme the old version. there is no know reason to use it now

Reply:
removed